docs/use-cases/azure.md

```diff
@@ -0,0 +1,113 @@
+fetching from or updating Azure Repos.
+
+Whether you're using the generated SSH deploy key or a Personal Access Token, the credentials used by
+Flux will need to be owned by an Azure DevOps User with access to the repo.
+Consider creating a machine-user and granting it granular permissions to access what's needed.
+This allows changing user access without affecting Flux.
+Since PAT's expire on Azure DevOps, using a machine-user's login password to authenticate with HTTPS and `libgit2`
+can be a good option that avoids the need to renew the credential while also having the benefit of more granular permissions.
+
+## Helm Repositories on Azure Container Registry
+
+The Flux `HelmRepository` object currently supports [Chart Repositories](https://helm.sh/docs/topics/chart_repository/)
+as well as fetching `HelmCharts` from paths in `GitRepository` sources.
+
+Azure Container Registry has a sub-command ([`az acr helm`](https://docs.microsoft.com/en-us/cli/azure/acr/helm)) for working with
+ACR-Hosted Chart Repositories, but it is deprecated.
+If you are using these deprecated Azure Chart Repositories, you can use Flux `HelmRepository` objects with them.
+
+[Newer ACR Helm documentation](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-helm-repos) suggests
+using ACR as an experimental [Helm OCI Registry](https://helm.sh/docs/topics/registries/).
+This will not work with Flux, because using Charts from OCI Registries is not yet supported.
+
+## Secrets Management with SOPS and Azure Key Vault
+
+You will need to create an Azure Key Vault and bind a credential such as a Service Principal or Managed Identity to it.
+If you want to use Managed Identities, install or enable [AAD Pod Identity](#aad-pod-identity).
+
+Patch kustomize-controller with the proper Azure credentials, so that it may access your Azure Key Vault, and then begin
+committing SOPS encrypted files to the Git repository with the proper Azure Key Vault configuration.
+
+See the [Mozilla SOPS Azure Guide](../guides/azure.md) for further detail.
+
+## Image Updates with Azure Container Registry
+
+You will need to create an ACR registry and bind a credential such as a Service Principal or Managed Identity to it.
+If you want to use Managed Identities, install or enable [AAD Pod Identity](#aad-pod-identity).
+
+You may need to update your Flux install to include additional components:
+```sh
+flux install \
+  --components-extra="image-reflector-controller,image-automation-controller" \
+  --export > ./clusters/my-cluster/flux-system/gotk-components.yaml
+```
+
+Follow the [Image Update Automation Guide](../guides/image-update.md) and see the
+[ACR specific section](../guides/image-update.md#azure-container-registry) for more details.
+
+Your AKS cluster's configuration can also be updated to allow the kubelets to pull images from ACR without ImagePullSecrets
```

Comment:
Can there be a link to documentation on how this can be done?

Reply:
Yep! Added